doc/user/content/release-notes.md

```diff
@@ -52,34 +52,42 @@ Wrap your release notes at the 80 character mark.
 - Support [`CREATE TABLE`](/sql/create-table), [`DROP TABLE`](/sql/drop-table),
   [`INSERT`](/sql/insert) and [`SHOW CREATE TABLE`](/sql/show-create-table).
 
-- Restore support for specifying multiple Kafka broker addresses in
-  [Kafka sources](/sql/create-source/avro-kafka/) {{% gh 3986 %}}.
-
-  This fixes a regression introduced in v0.4.1.
-
-- Remove the `max_timestamp_batch_size`
-  [`WITH` option](https://materialize.io/docs/sql/create-source/avro-kafka/#with-options)
-  from sources. Materialize now automatically selects the optimal batch size.
-  **Backwards-incompatible change.**
-
 <span id="v0.4.2"></span>
 ## v0.4.2 (Unreleased)
 
-- Output [SHOW COLUMNS](/sql/show-columns/) in the same order as the source {{% gh 4084 %}}.
+- Remove the `max_timestamp_batch_size` [`WITH`
+  option](/sql/create-source/avro-kafka/#with-options) from sources. Materialize
+  now automatically selects the optimal batch size. **Backwards-incompatible
+  change.**
+
+- Restore support for specifying multiple Kafka broker addresses in [Kafka
+  sources](/sql/create-source/avro-kafka/) {{% gh 3986 %}}.
+
+  This fixes a regression introduced in v0.4.1.
 
-- Improved memory utilization:
+- Sort the output of [`SHOW COLUMNS`](/sql/show-columns/) by the order in which
```

Comment:
nitpick: i just realized that `SHOW COLUMNS` is a link (via hover), whereas VALUES isn't -- maybe we should make "output of SHOW COLUMNS" all a link?

Reply:
Going to leave as is because the link is just to the page on `SHOW COLUMNS`, not to the section that describes the output of `SHOW COLUMNS`, and this is standard for how we link things in the docs. Would be great to get some docs for `VALUES` though so we have somewhere to link to for that! In general any release note that doesn't link to somewhere in the docs is a sign of missing docs.